### PR TITLE
TH-1062 | Add aluemeri to division blocklist

### PR DESCRIPTION
**Pull request by Sartonon**
## Description :sparkles:

- Delete Aluemeri from area dropdown
- Add test for the division hook

## Additional notes :spiral_notepad:

### CODE REVIEW

File: src/hooks/useDivisionOptions.tsx [hunk 1]
```diff
@@ -10,15 +14,20 @@ type DivisionOption = {
...
   const { data: neighborhoodsData } = useNeighborhoodListQuery();
+  const neighborhoodList = neighborhoodsData?.neighborhoodList.data;
+  const filteredNeighborhoodOptionList = neighborhoodList?.filter((option) =>
+    DIVISION_BLOCKLIST.every((blockOption) => blockOption !== option.id)
+  );
+
+  const neighborhoodToOptionValue = (neighborhood: Neighborhood) => ({
+    text: getLocalisedString(neighborhood.name, locale),
+    value: neighborhood.id,
+  });
```
Comment (nikomakela) [inline src/hooks/useDivisionOptions.tsx:25]: I think it is needless here to create a new variable that is used only once with a map function.

**Review by nikomakela — APPROVED** [review]
Looks approvable.

File: src/hooks/useDivisionOptions.tsx [hunk 2]
```diff
@@ -1,7 +1,11 @@
-import { useNeighborhoodListQuery } from '../generated/graphql';
+import sortBy from 'lodash/sortBy';
+
+import { Neighborhood, useNeighborhoodListQuery } from '../generated/graphql';
 import getLocalisedString from '../util/getLocalisedString';
 import useLocale from './useLocale';
 
+const DIVISION_BLOCKLIST = ['kaupunginosa:aluemeri'];
```
Comment (nikomakela) [inline src/hooks/useDivisionOptions.tsx:7]: If there ever comes more of these, I would move it to some const-file.

**Review by nikomakela — APPROVED** [review]

File: src/hooks/useDivisionOptions.tsx (hunk 1 above)
Comment (Sartonon) [reply src/hooks/useDivisionOptions.tsx:25]: Purpose is to only make it more readable, but maybe going too far?

File: src/hooks/useDivisionOptions.tsx (hunk 2 above)
Comment (Sartonon) [reply src/hooks/useDivisionOptions.tsx:7]: Hmmm maybe, but I think this is only going to be used in this helper hook and it only has only this one const. So is there  benefit by putting it to another file? Or do you mean like some global const file?

File: src/hooks/useDivisionOptions.tsx (hunk 1 above)
Comment (Sartonon) [reply src/hooks/useDivisionOptions.tsx]: Updated :)

**Pull request merged**